scripts/dockcross-manylinux-set-vars.sh

```diff
@@ -39,11 +39,11 @@ TARGET_ARCH=${TARGET_ARCH:=x64}
 
 # Specialized manylinux image tag to use for building.
 if [[ ${MANYLINUX_VERSION} == _2_28 && ${TARGET_ARCH} == x64 ]]; then
-  IMAGE_TAG=${IMAGE_TAG:=20230106-1aeaea0}
+  IMAGE_TAG=${IMAGE_TAG:=20230926-9eb419c}
 elif [[ ${MANYLINUX_VERSION} == _2_28 && ${TARGET_ARCH} == aarch64 ]]; then
   IMAGE_TAG=${IMAGE_TAG:=2022-11-19-1b19e81}
 elif [[ ${MANYLINUX_VERSION} == 2014 ]]; then
-  IMAGE_TAG=${IMAGE_TAG:=20230106-1aeaea0}
+  IMAGE_TAG=${IMAGE_TAG:=20230926-9eb419c}
```

Comment:
For future reference, the associated dockcross changes are the following: https://github.com/dockcross/dockcross/compare/1aeaea0..9eb419c

Summary:
* CMake `3.24.2` -> `3.27.5`
* Ninja `1.11.0` -> `1.11.1`
* Git `2.36.1` -> `2.42.0`
* Manylinux `2.28`
  * Update toolset from `gcc-toolset-11` to `gcc-toolset-12`
  * `pypa/manylinux_2_28_x86_64:2022-11-19-1b19e81` ->  `quay.io/pypa/manylinux_2_28_x86_64:2022-11-28-5d13db4`
* Manylinux `2014`
  * `pypa/manylinux2014_x86_64:2022-11-28-5d13db4` -> `pypa/manylinux2014_x86_64:2023-08-27-bd7ad21`